README.md

```diff
@@ -113,3 +113,15 @@ go test ./kaleido
 
 Optionally use `go test -v ./kaleido` to view
 all test logs.
+
+## Updating On-chain Registry smart contracts
```

Comment:
The instructions for updating the registry smart contracts can only be performed by someone with access to `photic-identity-smart-contracts`, which is limited to Kaleido employees since it is a private repo. I added a note to that section in the README to indicate that it is a Kaleido admin operation.